ansible/deploy.yml

```diff
@@ -27,7 +27,6 @@
   # Facts are already collected in the previous playbook
   gather_facts: no
   max_fail_percentage: 25
```

Comment:
As no other item to run after this, this line is now unused?

Reply:
It still should fail and stop if 25% of the batch fails. We just removed the batch configuration, now the batch = all the hosts